### ISSUE TITLE
Empty priority creates a funky /etc/profiles.d/-foo.sh file

### ISSUE THREAD

**Issue by benpillet** [example42/puppet-profile#4]
## Expected Behavior

file should be something like 10-foo.sh
## Actual Behavior

file is -foo.sh
## Steps to Reproduce the Problem
1. leave priority empty
2. created file has a funky start character

**Comment by alvagante** [comment]
Yes, that's a silly thing. Problem now is that however I can the profile::script define now to fix it it would create a new file on existing systems, with the same content of the -filename one.
I'm actually undecided on what to do:
- Fix and break backwards compatibility (eventually bumping a new major version only for this reason)
- Leave as is
